sdk/core/Azure.Core/src/ResponseHeaders.cs

```diff
@@ -136,5 +157,12 @@ public bool Contains(string name)
         {
             return _response.ContainsHeader(name);
         }
+
+        private class ContentLengthOverflowException : OverflowException
```

Comment:
Out of curiosity - why the derived exception type here?

Reply:
Yeah, good question.  I don't think it adds much, seeing it now.  Will remove.